llvm/lib/Analysis/ModuleSummaryAnalysis.cpp

```diff
@@ -635,7 +635,8 @@ static void computeFunctionSummary(
                               HasIndirBranchToBlockAddress || HasIFuncCall;
   GlobalValueSummary::GVFlags Flags(
       F.getLinkage(), F.getVisibility(), NotEligibleForImport,
-      /* Live = */ false, F.isDSOLocal(), F.canBeOmittedFromSymbolTable());
+      /* Live = */ false, F.isDSOLocal(), F.canBeOmittedFromSymbolTable(),
+      /*ImportType=*/GlobalValueSummary::ImportKind::Definition);
```

Comment:
nit: I think you removed from the comment from other cases in this file already, but now that it's an enum and not "false" the comment seems less important 

same in "makeDummyFunctionSummary" in the ModuleSummaryIndex